packages/moonstone/styles/colors.less

```diff
@@ -127,6 +127,7 @@
 @moon-header-full-bleed-border-color: fade(@moon-white, 25%);
 @moon-header-input-bg-color:          @moon-spotlight-color;
 @moon-header-input-text-color:        @moon-spotlight-text-color;
+@moon-header-controls-border-color:   #383838;
```

Comment:
Noticed that we used an specific value here and the `@moon-border-color` variable above and wanted to verify it was by design.

Reply:
Yes, it's intentional. The Header border is not using our standard colors, I believe for perceptual reasons.